releng/platform-definitions/platform-definition-2023-12/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2023, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Same as the platform definition, doesn't need the 2023 and you brought in the Datadog line as a heads up.

Reply:
> Same as the platform definition, doesn't need the 2023 and you brought in the Datadog line as a heads up.

Changed the copyright year.